src/biokbase/narrative/jobs/jobcomm.py

```diff
@@ -76,8 +74,21 @@ def __init__(self, rq: dict):
         self.request = self.rq_data.get("request_type")
         if self.request is None:
             raise ValueError("Missing request type in job channel message!")
-        self.job_id = self.rq_data.get("job_id")
-        self.job_id_list = self.rq_data.get("job_id_list")
+        if "job_id" in self.rq_data and "job_id_list" in self.rq_data:
```

Comment:
Add check for mutual exclusivity of job_id and job_id_list in constructor as well. Previously only in `JobRequest.translate`